driver/src/orderbook/streamed/updating_orderbook.rs

```diff
@@ -64,6 +80,14 @@ impl UpdatingOrderbook {
             .past_events(BlockNumber::Number(from_block.into()), to_block)
             .await?;
         self.handle_events(context, events, from_block).await?;
+        if self.filestore.is_some() {
+            if let Err(write_error) = context
+                .orderbook
+                .write_to_file(&self.filestore.as_ref().unwrap())
```

Comment:
What are your thoughts about not doing this on the critical path (but as a background task instead)? How long does writing the file to disk take? Probably not long compared to fetching the data, but it still feels a bit weird to do this synchronously when the rest of the system is desperately waiting for orderbook data.

Also, is there value in only writing if it's dirty (not in this PR).

Reply:
I expect this write to take less than a second unless there is something weird going on with kubernetes. Still we can move it into a thread in the future but it let's not do it in this PR.

Reply:
> Also, is there value in only writing if it's dirty (not in this PR).

Will create an issue here.. but if we define "dirty" as out of sync, then it would be consider always dirty at this point.